### PR TITLE
Add --log-driver=none to irssi run command.

### PR DESCRIPTION
**Pull request by psftw**

### CODE REVIEW

File: irssi/content.md [hunk 1]
```diff
@@ -18,26 +18,22 @@ On a Linux system, build and launch a container named `my-running-irssi` like th
...
 You omit `/etc/localtime` on Mac OS X because `boot2docker` doesn't use this file.
 
-Of course, you can name your image anything you like. In Docker 1.5 you can also use the `--read-only` mount flag. For example, on Linux:
-
-```console
-$ docker run -it --name my-running-irssi -e TERM -u $(id -u):$(id -g) \
-    --read-only -v $HOME/.irssi:/home/user/.irssi \
-    -v /etc/localtime:/etc/localtime \
-    irssi
-```
```
Comment (tianon) [inline irssi/content.md]: Why'd you axe this section?

Comment (psftw) [inline irssi/content.md]: what value does it add?

**Comment by psftw** [comment]
attempt to fix CI by actually using markdownfmt.sh

**Comment by psftw** [comment]
alright, one more time without README.md getting updated.  maybe I'll learn what I'm doing one day.

File: irssi/content.md (hunk 1 above)
Comment (tianon) [inline irssi/content.md]: I think the idea was to point out that `~/.irssi` is the _only_ place irssi tries to mutate by default.

**Comment by tianon** [comment]
LGTM

cc @jfrazelle

**Comment by jfrazelle** [comment]
nice LGTM

**Pull request merged**